tests_integration/__tests__/loglevel.js

```diff
@@ -2,51 +2,39 @@
 
 const runPrettier = require("../runPrettier");
 
-const threshold = 20;
-
-expect.addSnapshotSerializer({
-  test: value =>
-    typeof value === "string" &&
-    value.split("\n").some(line => line.length > threshold),
-  print: (value, serializer) =>
-    serializer(
-      value
-        .split("\n")
-        .map(
-          line =>
-            line.length > threshold
-              ? line.slice(0, threshold - 3) + "..."
-              : line
-        )
-        .join("\n")
-    )
+test("do not show logs with --loglevel silent", () => {
+  runPrettierWithLogLevel("silent", null);
 });
 
-describe("do not show logs with --loglevel silent", () => {
-  runPrettierWithLogLevel("silent");
+test("do not show warnings with --loglevel error", () => {
+  runPrettierWithLogLevel("error", ["[error]"]);
 });
 
-describe("do not show warnings with --loglevel error", () => {
-  runPrettierWithLogLevel("error");
+test("show errors and warnings with --loglevel warn", () => {
+  runPrettierWithLogLevel("warn", ["[error]", "[warn]"]);
 });
 
-describe("show errors and warnings with --loglevel warn", () => {
-  runPrettierWithLogLevel("warn");
+test("show all logs with --loglevel debug", () => {
+  runPrettierWithLogLevel("debug", ["[error]", "[warn]", "[debug]"]);
 });
 
-describe("show all logs with --loglevel debug", () => {
-  runPrettierWithLogLevel("debug");
-});
-
-function runPrettierWithLogLevel(logLevel) {
-  runPrettier("cli/loglevel", [
+function runPrettierWithLogLevel(logLevel, patterns) {
+  const result = runPrettier("cli/loglevel", [
     "--loglevel",
     logLevel,
     "--unknown-option",
     "--parser",
     "unknown-parser",
     "not-found.js"
-  ]).test({
-    status: "non-zero"
-  });
+  ]);
+
+  expect(result).not.toEqual(0);
+
+  if (patterns) {
+    patterns.forEach(pattern => {
+      expect(result.stderr).toMatch(pattern);
+    });
+  } else {
+    expect(result.stderr).toMatch(/\s*/);
```

Comment:
What is this supposed to do? _Any_ string matches `/\s*/`, right?

Reply:
Oops, I forgot to add `^`+`$`